frontend/src/employee-frontend/components/finance-basics/ServiceNeedsSection.tsx

```diff
@@ -14,29 +14,36 @@ import { renderResult } from '../async-rendering'
 
 import PlacementTypeItem from './PlacementTypeItem'
 import { serviceNeedsQuery } from './queries'
+import { useBoolean } from 'lib-common/form/hooks'
+import {PlacementType} from "../../../lib-common/generated/api-types/placement";
 
 export default React.memo(function ServiceNeedsSection() {
   const { i18n } = useTranslation()
 
-  const [open, setOpen] = useState(true)
-  const toggleOpen = useCallback(() => setOpen((isOpen) => !isOpen), [setOpen])
+  const [open, useOpen] = useBoolean(false)
```

Comment:
just FYI: you can avoid coming up with a name for the second argument (useOpen) by destructuring further:
`const [open, { toggle: toggleOpen }] = useBoolean(false)`